fhevm/operators_arithmetic_gas.go

```diff
@@ -2,6 +2,59 @@ package fhevm
 
 import "encoding/hex"
 
+func sgxAddSubRequiredGas(environment EVMEnvironment, input []byte) uint64 {
```

Comment:
For clarity, I think we can create a separate `sgx/` package, and put everything SGX-related there.